### PR TITLE
refactor(components): [popconfirm] changed button type

### PR DESCRIPTION
**Pull request by ryuhangyeong**
## Work History

1. `changed button type`

    Since `text` already exists in `buttonTypes` , I think you can use `buttonTypes`.

2. `changes to attribute kebab case, add accepted values`

    changed attributes to kebab case and added acceptable values ​​for button type.

---

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: docs/en-US/component/popconfirm.md [hunk 1]
```diff
@@ -39,18 +39,18 @@ popconfirm/trigger-event
...
+| title               | Title                                                                               | String             | —                                                             | —              |
+| confirm-button-text | Confirm button text                                                                 | String             | —                                                             | —              |
+| cancel-button-text  | Cancel button text                                                                  | String             | —                                                             | —              |
+| confirm-button-type | Confirm button type                                                                 | String             | primary / success / warning / danger / info / <del>text</del> | primary        |
+| cancel-button-type  | Cancel button type                                                                  | String             | primary / success / warning / danger / info / <del>text</del> | text           |
```
Comment (sxzz) [inline docs/en-US/component/popconfirm.md:48]: `text` here is not a deprecated text, so please remove `<del>`

Comment (ryuhangyeong) [reply docs/en-US/component/popconfirm.md]: resolve! thank you for you detail review!

Comment (ryuhangyeong) [reply docs/en-US/component/popconfirm.md]: resolve! thank you for detail review!

by the way, there is an error in the test of 'time-picker' that I did not modify. 🤔

File: docs/en-US/component/popconfirm.md [hunk 2]
```diff
@@ -39,18 +39,18 @@ popconfirm/trigger-event
...
+| ------------------- | ----------------------------------------------------------------------------------- | ------------------ | ------------------------------------------------------------- | -------------- |
+| title               | Title                                                                               | String             | —                                                             | —              |
+| confirm-button-text | Confirm button text                                                                 | String             | —                                                             | —              |
+| cancel-button-text  | Cancel button text                                                                  | String             | —                                                             | —              |
+| confirm-button-type | Confirm button type                                                                 | String             | primary / success / warning / danger / info / <del>text</del> | primary        |
```
Comment (sxzz) [inline docs/en-US/component/popconfirm.md:47]: `<del>` stills here.

File: docs/en-US/component/popconfirm.md (hunk 1 above)
Comment (sxzz) [reply docs/en-US/component/popconfirm.md]: Sometimes, CI will failed with unknown reason. (there's a problem in our unit test). Just re-run the tests.

File: docs/en-US/component/popconfirm.md (hunk 2 above)
Comment (ryuhangyeong) [reply docs/en-US/component/popconfirm.md]: done! Sorry for making you work twice.

**Review by sxzz — APPROVED** [review]

**Pull request merged**